### PR TITLE
gennodejs: 2.0.1-0 in 'lunar/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by IanTheEngineer**
Increasing version of package(s) in repository `gennodejs` to `2.0.1-0`:

- upstream repository: https://github.com/RethinkRobotics-opensource/gennodejs.git
- release repository: https://github.com/RethinkRobotics-release/gennodejs-release.git
- distro file: `lunar/distribution.yaml`
- bloom version: `0.5.23`
- previous version for package: `null`

## gennodejs

```
* Fix an issue uncovered in checking fixed size message fields
* Contributors: Chris Smith
```

### CODE REVIEW

**Comment by dirk-thomas** [comment]
Can you please also add a `source` entry so that users can easily get the sources using the `rosinstall_generator`. If you don't want a devel job for the repo you can add `test_commits: false` to the source entry.

**Comment by IanTheEngineer** [comment]
Yep! Should be all good now. Lemme know if there's anything else we need to do for the release :)

File: lunar/distribution.yaml [hunk 1]
```diff
@@ -59,6 +59,22 @@ repositories:
...
+      type: git
+      url: https://github.com/RethinkRobotics-opensource/gennodejs.git
+      version: kinetic-devel
+    release:
+      tags:
+        release: release/lunar/{package}/{version}
+      url: https://github.com/RethinkRobotics-release/gennodejs-release.git
+      version: 2.0.1-0
+    source:
+      test_pull_requests: true
```
Comment (dirk-thomas) [inline lunar/distribution.yaml]: Have you made sure the buildfarm has access to your repo / org unit (see http://wiki.ros.org/buildfarm/Pull%20request%20testing)?

Comment (IanTheEngineer) [inline lunar/distribution.yaml]: Excellent, thanks for that resource! I just added the webhook for issue_comment and pull_request, so we should be good to go!

Comment (dirk-thomas) [inline lunar/distribution.yaml]: I haven't seen that you gave permission to the @ros-pull-request-builder yet. So the PR integration might not be complete yet.

**Comment by dirk-thomas** [comment]
Thanks for the quick PR.

**Pull request merged**